python/src/functions/generate_treasury_report.py

```diff
@@ -215,16 +223,24 @@ def process_event(payload: ProjectLambdaPayload, logger):
             upload_generated_file_to_s3(
                 client=s3_client,
                 bucket=os.environ["REPORTING_DATA_BUCKET_NAME"],
-                key=f"{file_destination_prefix}.csv",
+                key=get_generated_output_file_key(
+                    file_type=OutputFileType.CSV,
+                    project=project_use_code,
+                    organization=organization,
+                ),
                 file=csv_file,
             )
         # Store project_id_agency_id to row number in a json file
         with tempfile.NamedTemporaryFile("w") as json_file:
-            json_file = json.dump(project_agency_id_to_row_map)
```

Comment:
@vshia would be great to get your input here. I updated this based on typing errors I noticed. However I wasn't sure if this breaks any of the existing functionality.